09-echo_data.ipynb

```diff
@@ -18,10 +18,11 @@
     "from res.sequential_tasks import EchoData\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.    #   [[S11 S12...S1N], [S22 S22...S2N], ..., [SM1 SM2...SMN]]
</pre><p>The first <code>S22</code> should be <code>S21</code> instead.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Atcold/pytorch-Deep-Learning/pull/830/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Thanks! Done.